src/components/core/Grid/Grid.md

```diff
@@ -0,0 +1,24 @@
+Grid is a [Box](#/Primitives?id=box) with the CSS grid layout applied.
```

Comment:
I'm not super convinced that it's necessary ;-)

Reply:
I think it makes the code easier to read and communicate the intent. Same as with `Flex` imo.

Reply:
You almost convinced me ;-)